### PR TITLE
Issue#409 swap out json editor for form

### PR DESCRIPTION
**Pull request by Mkohm**
Resolves https://github.com/GlobalDigitalLibraryio/issues/issues/409

### CODE REVIEW

File: packages/admin-frontend/components/edit_book/EditBookForm.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                      </>
+                    )}
+                  />
+                </Grid>
+
+                <Grid item xs>
+                  <Field
+                    name="publishingStatus"
+                    render={({ input }) => (
+                      <>
```
Comment (alexanbj) [inline packages/admin-frontend/components/edit_book/EditBookForm.js]: The fragment wrapping the select here is redudant. We should also label the field.

File: packages/admin-frontend/components/edit_book/EditBookForm.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                      </>
+                    )}
+                  />
+                </Grid>
+
+                <Grid item xs>
+                  <Field
+                    name="pageOrientation"
+                    render={({ input }) => (
+                      <>
```
Comment (alexanbj) [inline packages/admin-frontend/components/edit_book/EditBookForm.js]: The fragment wrapping the select here is redudant. We should also label the field.

File: packages/admin-frontend/components/edit_book/EditBookForm.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            </Link>
+          </Typography>
+        )}
+        <Form
+          initialValues={book}
+          onSubmit={this.handleSubmit}
+          validate={handleValidate}
+          render={({ handleSubmit, pristine, form, invalid }) => (
+            <form>
+              <Grid container spacing={24} direction="column">
```
Comment (alexanbj) [inline packages/admin-frontend/components/edit_book/EditBookForm.js]: Suggestion: Simplify spacing by removing the grid thingy and set margin on the TextFields instead? See https://material-ui.com/api/text-field/

File: packages/admin-frontend/style/media.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+// @flow
```
Comment (alexanbj) [inline packages/admin-frontend/style/media.js]: This isn't in use. Should be deleted

Comment (Mkohm) [reply packages/admin-frontend/style/media.js]: fixed

File: packages/admin-frontend/components/edit_book/EditBookForm.js (hunk 1 above)
Comment (Mkohm) [reply packages/admin-frontend/components/edit_book/EditBookForm.js]: fixed

File: packages/admin-frontend/components/edit_book/EditBookForm.js (hunk 2 above)
Comment (Mkohm) [reply packages/admin-frontend/components/edit_book/EditBookForm.js]: fixed

File: packages/admin-frontend/components/edit_book/EditBookForm.js (hunk 3 above)
Comment (Mkohm) [reply packages/admin-frontend/components/edit_book/EditBookForm.js]: fixed

**Pull request merged**